products/chef-infra-server.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Chef Infra Server
+category: server-app
+iconSlug: chef
+tags: erlang-runtime ruby-runtime
+permalink: /chefinfraserver
+alternate_urls:
+-   /chefserver
+versionCommand: chef-server-ctl version
+releasePolicyLink: "https://docs.chef.io/versions/"
+changelogTemplate: "https://docs.chef.io/release_notes_server/#__LATEST__"
+
+activeSupportColumn: true
+releaseDateColumn: true
+
+auto:
+- custom: true
+
+# eol(x) = releaseDate(x+2) > confirm EOL date here https://docs.chef.io/versions/
+# support(x) = releaseDate(x+1)
+releases:
+-   releaseCycle: "15"
+    releaseDate: 2022-06-13
+    support: true
+    eol: false
+    latest: "15.9.20"
+
+-   releaseCycle: "14"
+    releaseDate: 2020-06-08
+    support: 2022-06-13
+    eol: false
+    latest: "14.16.19"
+
+-   releaseCycle: "13"
+    releaseDate: 2019-05-31
+    support: 2020-06-08
+    eol: 2021-06-30
+    latest: "13.2.0"
+
+-   releaseCycle: "12"
+    releaseDate: 2014-11-24
+    support: 2019-05-31
+    eol: 2020-12-31
+    latest: "12.19.31"
+
+-   releaseCycle: "11"
+    releaseDate: 2013-09-30
+    support: 2014-11-24
+    eol: 2018-12-31
+    latest: "11.1.7"
+
+---
+
+> [Chef Infra Server](https://docs.chef.io/server/) is an open-source platform that complements Chef Infra Client.
+> It serves as a central hub for configuration data, providing scalable infrastructure management. 
+
+Chef Infra Server generally follows an N-1 support strategy  
+(meanings that the 2 most recent major versions receive security and bug fixes)
```

Comment:
Could you give a bit more details on the two support phases explained on https://docs.chef.io/versions/#lifecycle-definitions (example of such documentation: https://endoflife.date/nodejs) ? I quickly went through https://docs.chef.io/versions/#lifecycle-definitions and I did not find the rule `support(x) = releaseDate(x+1)` (e.g. when a release become deprecated).

Reply:
TLTE, I was relying on bad data.
For Chef Infra Server, only the latest release is supported
I fix that

Reply:
Based on [Here](https://www.chef.io/service-level-agreement?_gl=1*1xlphvy*_ga*ODEyNjY0NTc4LjE3MDM3MTUwMDg.*_ga_9JSNBCSF54*MTcwODQ0OTk4Mi4xNS4xLjE3MDg0NTI2MzcuNTkuMC4w*_gcl_au*MTYyMjE3NTI1MS4xNzAzNzE1MDk4)

“Deprecated” means Supported Applications that are supported by Chef but for which a newer version or alternative Supported Applications are available and recommended for Chef customer use. Chef no longer provides scheduled releases for Deprecated Software but may produce ad hoc Updates in response to customer issues and/or security vulnerabilities at the sole discretion of Chef. Deprecated Software is targeted for End of Life but a timeline has not yet been announced for End of Life.


And here, https://docs.chef.io/versions/#deprecated-products-and-versions

v15 is GA
v14 is "deprecated"
V13 and below are EOL

Based on that, it seems coherent to me, but maybe I don't understand where you're going with this?
```
 eol(x) = releaseDate(x+2) > confirm EOL date here https://docs.chef.io/versions/
 support(x) = releaseDate(x+1)
```

I updated the desc as follow, let me know.

The only actively maintained version [is the latest](https://docs.chef.io/versions/#supported-commercial-distributions). Release N-1 are "deprecated" [and can only receive security updates](https://www.chef.io/service-level-agreement?_gl=1*1xlphvy*_ga*ODEyNjY0NTc4LjE3MDM3MTUwMDg.*_ga_9JSNBCSF54*MTcwODQ0OTk4Mi4xNS4xLjE3MDg0NTI2MzcuNTkuMC4w*_gcl_au*MTYyMjE3NTI1MS4xNzAzNzE1MDk4). Releases N-2 [and below are EOL](https://docs.chef.io/versions/#end-of-life-eol-products).